### DISCUSSION TITLE
upper case groups generated by dex server

### DISCUSSION THREAD

**Discussion by brackend** [argoproj/argo-workflows#7522] [Q&A] [answered]
We have enabled dex alongside Argocd in helm files.  Dex handles the oidc flows with our IDP.  But by default Dex is converting the groups to upper case.  You could use upper case when setting permissons in Argocd.  But for various reasons our preferred option would be to change the dex config to preserve the case - ie, groups are not changed from their original case.

Any help with this very much welcome.

**Comment by terrytangyuan** [comment]
You might want to ask in Argo CD repo instead.

**Comment by brackend** [comment, marked as answer]
Thank you.  Turned out that our idp was converting to upper case before
sending to dex/argocd
